.docs/content/1.concepts/7.adaptors.md

```diff
@@ -9,6 +9,8 @@ It keeps track of the state of the computations and data managed by ArmoniK.
 
 An interface lies between each service and the other components of Core. The adaptors provide implementations of the interfaces. The interface abstracts away all the complexities and the details of the underlying service (queue, database or object storage), and provides the features of the service that Core components may need. A NuGet package [ArmoniK.Core.Base](https://www.nuget.org/packages/ArmoniK.Core.Base/) is generated, it contains the interfaces and classes needed for the implementation of a custom adaptor.
```

Comment:
An adaptor provides a way to connect ArmoniK.Core to other services. To communicate, an interface lies between the queue and the other components of Core. The adaptor implements this interface. For example, a queue adaptor acts as a middle layer between the queue and the other components of Core.

Reply:
I removed the example because I want this section to be generic, not to be about queue adaptors in particular